libraries.json

```
@@ -13,53 +19,212 @@
 		"image": "https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/angular/angular.png",
 		"language": "JavaScript"
 	},
+	"BabylonJS": {
+		"imports": ["babylonjs"],
+		"technologies": ["WebGL"],
+		"description": "Powerful, beautiful, simple, open, web-based 3d at its best",
```

```suggestion
		"description": "Powerful, beautiful, simple, open, web-based 3D at its best",
```

nit: the d in the 3D is uppercase